deploy/complete/terraform/schema.yaml

```diff
@@ -119,7 +133,6 @@ variables:
   num_pool_workers:
     type: integer
     title: "Number of Worker Nodes"
-    default: 1
```

Comment:
Why remove the defaults ?

Reply:
Only necessary defaults need to be on the schema. Defaults should come from variables always as possible.